FC-0068: Docs Cleanup

jswope00 opened the pull request:
Mainly two items:
- Fix RST formatting after splitting files up during migration
- Add "See Also" links

openedx-webhooks commented:
Thanks for the pull request, @jswope00!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

sarina commented on source/educators/concepts/data/learner_data.rst:
```diff
@@ -45,9 +44,9 @@ For information about course enrollment, see :ref:`Enrollment`.
 ===========================
 
-When learners create, or register, their user accounts, they select a public
+When learners create or register, their user accounts, they select a public
```
```suggestion
When learners create or register their user accounts, they select a public
```

sarina commented on source/educators/concepts/releasing-course/beta_testing.rst:
```diff
@@ -124,7 +124,7 @@ Beta testers should interact with everything in the course.
 * Complete all problems.
 
-Beta testers can use a desktop computer to access an edX website, the edX
+Beta testers can use a desktop computer to access an your course website, the
```
```suggestion
Beta testers can use a desktop computer to access your course website, the
```

sarina commented on source/educators/how-tos/course_development/configure_resources.rst:
```diff
@@ -32,17 +31,17 @@ To change how the discussion experience is configured, follow these steps:
 
 The discussion configuration experience is comprised of two steps: provider selection and provider configuration.
-By default, courses are created with the edX discussion experience and its default configuration.
+By default, courses are created with the discussion experience and its default configuration.
```
```suggestion
By default, courses are created with the Open edX discussion experience and its default configuration.
```

There are multiple different types of discussion you can use, like YellowDig or piazza, so good to specify the name of the default.

sarina commented on source/educators/how-tos/course_development/configure_resources.rst:
```diff
@@ -32,17 +31,17 @@ To change how the discussion experience is configured, follow these steps:
 **Step 1: Provider Selection**
 
-In this step, educators can select to use the default edX discussion experience or LTI based integrations with other providers.
+In this step, educators can select to use the default discussion experience or LTI based integrations with other providers.
```
```suggestion
In this step, educators can select to use the default Open edX discussion experience or LTI based integrations with other providers.
```

sarina commented on source/educators/how-tos/course_development/configure_resources.rst:
```diff
@@ -32,17 +31,17 @@ To change how the discussion experience is configured, follow these steps:
 **Step 2: Configuration Options**
 
-Each provider supports its own set of features depending on the discussion features they support. The edX discussion application has various settings and controls that are described below.
+Each provider supports its own set of features depending on the discussion features they support. The discussion application has various settings and controls that are described below.
```
```suggestion
Each provider supports its own set of features depending on the discussion features they support. The default Open edX discussion application has various settings and controls that are described below.
```

sarina commented on source/educators/how-tos/course_development/configure_resources.rst:
```diff
@@ -81,24 +80,24 @@ To change access to the course wiki, follow these steps.
 
 The **Enable public wiki** access wiki setting is disabled by default, meaning that only course team members and
-enrolled learners can see the course wiki. If you enable this setting, then any registered edX user can access the
+enrolled learners can see the course wiki. If you enable this setting, then any registered user can access the
```
```suggestion
enrolled learners can see the course wiki. If you enable this setting, then any registered user on your site can access the
```

sarina commented on source/educators/how-tos/course_development/configure_resources.rst:
```diff
@@ -81,24 +80,24 @@ To change access to the course wiki, follow these steps.
-enrolled learners can see the course wiki. If you enable this setting, then any registered edX user can access the
+enrolled learners can see the course wiki. If you enable this setting, then any registered user can access the
 course wiki, even if they are not enrolled in your course. However, public users would have to explicitly navigate
 to your wiki via the edX-wide wiki structure, or a link that has been provided to them.
```
```suggestion
to your wiki via the site-wide wiki structure, or a link that has been provided to them.
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
+ :ref:`Course Outline` (concept)
```
```suggestion
 :ref:`Course Outline` (concept)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
+ :ref:`Course Outline` (concept)
+ :ref:`Creating a New Course` (how-to)
```
```suggestion
 :ref:`Creating a New Course` (how-to)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Getting Started with Course Content Development` (reference)
+ :ref:`Course Outline` (concept)
+ :ref:`Creating a New Course` (how-to)
+ :ref:`Create the Course About Page` (how-to)
```
```suggestion
 :ref:`Create the Course About Page` (how-to)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Hide a Subsection from Students` (how-to)
+ :ref:`Add Course Metadata` (how-to)
+ :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
+ :ref:`Resources for edx.org Course Teams <Resources for edx org Course Teams>` (reference)
```
```suggestion
 :ref:`Resources for edx.org Course Teams <Resources for edx org Course Teams>` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Create a Section` (how-to)
+ :ref:`Hide a Subsection from Students` (how-to)
+ :ref:`Add Course Metadata` (how-to)
+ :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
```
```suggestion
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Developing Course Subsections` (reference)
+ :ref:`Create a Section` (how-to)
+ :ref:`Hide a Subsection from Students` (how-to)
+ :ref:`Add Course Metadata` (how-to)
```
```suggestion
 :ref:`Add Course Metadata` (how-to)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Developing Course Sections` (reference)
+ :ref:`Developing Course Subsections` (reference)
+ :ref:`Create a Section` (how-to)
+ :ref:`Hide a Subsection from Students` (how-to)
```
```suggestion
 :ref:`Hide a Subsection from Students` (how-to)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Publish Content from the Course Outline` (reference)
+ :ref:`Developing Course Sections` (reference)
+ :ref:`Developing Course Subsections` (reference)
+ :ref:`Create a Section` (how-to)
```
```suggestion
 :ref:`Create a Section` (how-to)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Modify Settings for Objects in the Course Outline` (reference)
+ :ref:`Publish Content from the Course Outline` (reference)
+ :ref:`Developing Course Sections` (reference)
+ :ref:`Developing Course Subsections` (reference)
```
```suggestion
 :ref:`Developing Course Subsections` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Developing Your Course Outline` (reference)
+ :ref:`Modify Settings for Objects in the Course Outline` (reference)
+ :ref:`Publish Content from the Course Outline` (reference)
+ :ref:`Developing Course Sections` (reference)
```
```suggestion
 :ref:`Developing Course Sections` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Add Content in the Course Outline` (reference)
+ :ref:`Developing Your Course Outline` (reference)
+ :ref:`Modify Settings for Objects in the Course Outline` (reference)
+ :ref:`Publish Content from the Course Outline` (reference)
```
```suggestion
 :ref:`Publish Content from the Course Outline` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Understanding Your Course Outline` (reference)
+ :ref:`Add Content in the Course Outline` (reference)
+ :ref:`Developing Your Course Outline` (reference)
+ :ref:`Modify Settings for Objects in the Course Outline` (reference)
```
```suggestion
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Create the Course About Page` (how-to)
+ :ref:`Understanding Your Course Outline` (reference)
+ :ref:`Add Content in the Course Outline` (reference)
+ :ref:`Developing Your Course Outline` (reference)
```
```suggestion
 :ref:`Developing Your Course Outline` (reference)
 
```

sarina commented on source/educators/how-tos/course_development/create_edit_publish_subsections.rst:
```diff
@@ -110,4 +110,25 @@ To delete a subsection, follow these steps.
+ :ref:`Creating a New Course` (how-to)
+ :ref:`Create the Course About Page` (how-to)
+ :ref:`Understanding Your Course Outline` (reference)
+ :ref:`Add Content in the Course Outline` (reference)
```
```suggestion

 :ref:`Add Content in the Course Outline` (reference)
 
```

sarina commented on source/educators/how-tos/create_section.rst:
```diff
@@ -1,6 +1,32 @@
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course` (how-to)
 
 :ref:`Create a Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Subsection ` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

sarina replied:
Are you checking how pages render? You need spaces here otherwise they all render in a big block which is illegible. This is a common mistake in this PR and is why I want smaller changesets, so we can catch problems early.

jswope00 replied:
@srushe1 can you take a pass through all our pages and check if there are any remaining places where this is happening?

sarina commented on source/educators/how-tos/create_subsection.rst:
```diff
@@ -4,3 +4,25 @@ Create a Subsection
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)

 :ref:`Course Outline` (concept)
 
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)

  :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)

  :ref:`Add Course Metadata` (how-to)

  :ref:`Use a Section from a Course independently of the Course Outline` (how-to)

  :ref:`Resources for edx org Course Teams` (reference)
 :ref:`Resources for Open edX` (reference)

```

sarina commented on source/educators/how-tos/issue_certificates.rst:
```diff
@@ -94,5 +94,14 @@ To allow learners to download early certificates, you modify the
+
+  :ref:`Enable Badges in Course` (how-to)
+
+ :ref:`Reporting Certificate Data` (how-to)
```
```suggestion
  :ref:`Reporting Certificate Data` (how-to)
```

sarina commented on source/educators/how-tos/rerun_course.rst:
```diff
@@ -31,7 +31,3 @@ steps.
   while the process runs. The new course appears on your **My Courses**
   dashboard in Studio when configuration is complete.
 
-.. seealso::
```
Why delete this See Also?

jswope00 replied:
Seems like there was some kind of uncertainty about how to format the targets. In the deleted See Also section I am seeing a few targets that look like they are the same, just with different formatting. I'll take note of this doc and any others and make sure they get See Also's.

jswope00 replied:
This was a stub file that ended up getting merged into course_rerun.rst, since we won't be using stub reusable files much. Resolved the issue, since course_rerun.rst has proper See Also section.

sarina commented on source/educators/how-tos/set_up_course/add_metadata.rst:
```diff
@@ -17,4 +17,26 @@ Add Course Metadata
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
 :ref:`Course Outline` (concept)
 
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection ` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

sarina commented on source/educators/how-tos/set_up_course/edit_course_about.rst:
```diff
@@ -45,8 +44,7 @@ change the way that your course title and number appear in Studio and the LMS.
+To change the way that your course title appears in Studio and the LMS, follow these steps.
 
 .. note::
  EdX does not recommended that you do this if your course is live.
```
```suggestion
 It is not recommended that you do this if your course is live.
```

Did your search also search for instances of the string "EdX"?

jswope00 replied:
I think it was a manual read through each doc. I'll do a batch search.

sarina commented on source/educators/how-tos/set_up_course/set_schedule.rst:
```diff
@@ -40,13 +39,13 @@ follow these steps.
   The times that you set are in Coordinated Universal Time (UTC). You might
   want to verify that you have specified the times that you intend by using a
-  time zone converter such as `Time and Date Time Zone Converter`_.
+  time zone converter such as `Time and Date Time Zone Converter`.
```
The underscore is needet to make this a link.

```suggestion
  time zone converter such as `Time and Date Time Zone Converter`_.
```

sarina commented on source/educators/how-tos/update_rerun_course.rst:
```diff
@@ -97,3 +97,13 @@ for launch, see :ref:`Launch`.
 
```
Above in this file there is an `.. only:: Partners` stanza. These need to all be found and deleted.

jswope00 replied:
This file is under aulusneo's cleanup, but we had to touch it to change the name of a target. It'll get cleaned under aulusneo's ongoing work.

sarina commented on source/educators/references/course_development/add_edit_in_outline.rst:
```diff
@@ -128,4 +126,26 @@ You are prompted to confirm the deletion.
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

sarina commented on source/educators/references/course_development/course_number_guidelines.rst:
```diff
@@ -23,17 +23,23 @@ Course numbers have the following guidelines.
 
 * CS002x
 * BIO1.1x and BIO1.2x
 * 6.002.1x and 6.002.2x
```
Should we really recommend putting "x" at the end of every course number? Maybe something like

```suggestion
* CS101
* BI01.1 and BI01.2
* 6.002.1x and 6.002.2x
```

jswope00 replied:
Agree, that's not a naming format that is relevant anymore. Accepting the suggestion.

sarina commented on source/educators/references/course_development/course_outline.rst:
```diff
@@ -45,4 +45,24 @@ following message is visible.
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

jswope00 replied:
Resolved spacing issues in See Also section.

sarina commented on source/educators/references/course_development/course_subsections.rst:
```diff
@@ -178,3 +178,26 @@ the subsection has been released.
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

sarina commented on source/educators/references/course_development/develop_course_sections.rst:
```diff
@@ -106,3 +104,25 @@ A section can contain a unit that is hidden from learners and available to
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
 :ref:`Course Outline` (concept)
 
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

jswope00 replied:
Resolved spacing issues in See Also section.

sarina commented on source/educators/references/course_development/modify_outline_settings.rst:
```diff
@@ -24,4 +24,26 @@ circled for a section, a subsection, and two units.
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for Open edX Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

jswope00 replied:
Resolved spacing issues in See Also section.

sarina commented on source/educators/references/course_development/publish_from_outline.rst:
```diff
@@ -26,4 +25,26 @@ For more information, see the following topics.
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
 :ref:`Course Outline` (concept)
 
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for Open edX Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
 
```

jswope00 replied:
Resolved proper spacing issue.

sarina commented on source/educators/references/course_development/understanding_course_outline.rst:
```diff
@@ -34,4 +34,27 @@ When you expand a subsection, all units in the subsection are visible.
+.. seealso::
+ :class: dropdown
+
+ :ref:`Getting Started with Course Content Development` (reference)
```
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

sarina commented on source/educators/references/resources_for_course_teams.rst:
```diff
@@ -1,6 +1,8 @@
-###################################
+.. _Resources for edx org Course Teams: 
```
```suggestion
.. _Resources for Open edX Course Teams: 
```

sarina commented on source/educators/references/workflow.rst:
```diff
@@ -42,7 +42,7 @@ Before you begin, you should understand the building blocks of an edX course.
 ****************************************
 
-Once you understand the way edX courses are structured, you can start
+Once you understand the way courses are structured on Open edX®, you can start
```
```suggestion
Once you understand the way courses are structured on the Open edX® platform, you can start
```

sarina commented on source/educators/references/workflow.rst:
```diff
@@ -146,9 +146,32 @@ visibility:
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Course Outline` (concept)
```
```suggestion
 :ref:`Course Outline` (concept)
 
  :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
  
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners  <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for Open edX Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)  
 
```

jswope00 commented:
This PR is large, so I'm breaking down the work done and review requested. @sarina 

1. Fix RST formatting: In many files we adjusted RST formatting mostly by shifting headers (e.g. when we broke up a file and now an h2 needs to become an h1, etc). 
2. Adjust references to edx.org either by changing to Open edX, deleting the edx.org term, or modifying the snippet if it is an edx.org specific thing that doesn't exist in Open edX. 
3. Added 'seealso' tables in documents. And then, in later commits, ensured there were spaces between seealso list items so that they are rendered with line breaks. 
4. [This commit](https://github.com/openedx/docs.openedx.org/pull/649/commits/eba68aa321a950c4c97314b0b1e8acf50dcb0d93) probably needs the closest look, as it relates to accessibility. There were edx.org-specific notes that mentioned in accessible course content that I deleted. And some references to reaching out to edx.org support and accessibility services that I changed to recommending users submit GIthub issues.
